playbooks/collect_instance_statistics/collect_elasticsearch_data.yml

```diff
@@ -0,0 +1,59 @@
+---
+# A simple one-off playbook that gathers basic usage information from the configured logs server.
+# This is used by the `instance_statistics_csv` management command.
+
+- hosts: all
+  vars:
+    # This should be overridden to be more unique.
+    local_output_dir: /tmp/
+    remote_output_filename: /tmp/activity_report
+  tasks:
+    - name: Ensure necessary packages are installed
+      become: yes
+      apt:
+        name: ['python-setuptools', 'virtualenv']
+        state: present
+
+    - name: Copy elasticsearch_requirements.txt
+      copy:
+        src: elasticsearch_requirements.txt
+        dest: /tmp/elasticsearch_requirements.txt
+
+    - name: Install Elasticsearch packages
+      pip:
+        requirements: /tmp/elasticsearch_requirements.txt
+        virtualenv: /home/ubuntu/instance_activity_venv
```

Comment:
@pcockwell, not a good idea to hard-code the path. Can you make it a variable?

Reply:
Done in the next commit